code/src/components/ThoughtForm.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+
+const ThoughtForm = ({ newThought, onNewThoughtChange, onFormSubmit }) => {
+     
+    return (
```

Comment:
If you like, it would be nice to add so you can't send an empty messege! I did it like this:
before the return I added:
`const emptyTextarea = (value) => value.replace(/\s/g, "").length === 0;`
then in the return you can put this under line 20 where you disable:
`emptyTextarea(newThought)`
:)